src/test/kotlin/com/jay/racingcar/domain/RacingCarsTest.kt

```diff
@@ -6,22 +6,30 @@ import org.junit.jupiter.api.Test
 internal class RacingCarsTest {
     @Test
     fun `자동차들을 생성한다`() {
-        val racingCars = RacingCars.create(3, forwardStrategy())
+        val racingCars = RacingCars.create(Names(listOf("jay", "pobi", "honux")), forwardStrategy())
 
         assertThat(racingCars.racingCars.size).isEqualTo(3)
     }
 
     @Test
     fun `게임을 플레이하면 Strategy에 기반한 위치를 반환한다`() {
-        val racingCars = RacingCars.create(3, forwardStrategy())
+        val racingCars = RacingCars.create(Names(listOf("jay", "pobi", "honux")), forwardStrategy())
 
         racingCars.play()
 
         racingCars.racingCars.forEach {
-            assertThat(it.position).isEqualTo(1)
+            assertThat(it.getPosition()).isEqualTo(1)
         }
     }
 
+    @Test
+    fun `우승자의 목록을 반환한다`() {
+        val racingCars = RacingCars.create(Names(listOf("jay", "pobi", "honux")), forwardStrategy())
+        val winnerNames = racingCars.getWinners().map { it.getNameValue() }.toList()
```

Comment:
```suggestion
        val winnerNames = racingCars.getWinners().map { it.getNameValue() }.toList()
```
테스트 코드에 로직이 들어가 있네요.
테스트 코드에 로직이 있게 되면, 실제 코드의 로직이 변경되면 테스트 코드에도 변경을 해주어야 합니다.
이렇게 되면 테스트 코드가 로직을 제대로 검증하기가 어렵습니다.
테스트 코드에서는 최대한 로직을 배제해보면 어떨까요?